delfin/drivers/hpe/hpe_3par/alert_handler.py

```diff
@@ -102,7 +102,9 @@ def parse_alert(context, alert):
         try:
             alert_model = dict()
             # These information are sourced from device registration info
-            alert_model['alert_id'] = alert.get(AlertHandler.OID_MESSAGECODE)
+            alert_model['alert_id'] = (
```

Comment:
Why add '0' here

Reply:
Message in alarm document of 3Par_ The format of the code value is like this, in order to communicate with the message of the system_ Code values are consistent, so add '0' here

![image](https://user-images.githubusercontent.com/79956159/112103012-179e8080-8be4-11eb-9eea-36663ca7c45c.png)

Reply:
For example, what we found from the system is 2555934, which is 0x27001e after being converted to hexadecimal. Here you need to add a 0 to become 0x027001e, which is consistent with the 3par document.